.github/workflows/ci.yml

```diff
@@ -58,10 +58,16 @@ jobs:
           - "11.11-alpine"
           - "9.6-alpine"
           - "9.5-alpine"
+        include:
+          - elixirbase: "1.11.4-erlang-23.3.4.9-alpine-3.16.9"
+            postgres: "16.2-alpine"
+            pool_count: "4"
```

Comment:
I think this was not picked up? 🤔

Reply:
@greg-rychlewski do you know if this was meant to be picked up? Or only after we merge?

Reply:
@josevalim sorry I missed this. do you still want me to look at it?

Reply:
Oh, I think I know what it is. I think I need a default pool_count 1. Let me give it a try.